be/src/formats/parquet/file_reader.cpp

```diff
@@ -210,6 +210,23 @@ StatusOr<bool> FileReader::_filter_group(const tparquet::RowGroup& row_group) {
             if (discard) {
                 return true;
             }
+
+            // additional min/max check
+            if (min_chunk->columns()[0]->equals(0, *max_chunk->columns()[0], 0)) {
```

Comment:
why column data min/max should be equal?

Reply:
This is a special case that can be further filtered. 
If the min/max value is the same, we could use the runtime filter instead of the min/max conjuncts derived.